### PR TITLE
Whole site

### PR DESCRIPTION
**Pull request by jpdf00**
This is the first project in Microverse's HTML and CSS module. It consists of a reproduction of a page in the New York Times website. It displays the text, images and video of the original as well as some of the links. Advertisements and many of the interactive elements have been left out.

Built using HTML and CSS only.

### CODE REVIEW

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## Hi @jpdf00 and @German-Cobian 
Welcome to Microverse :tada: and congratulations on submitting your first project.
Your projects do not meet the requirements, as such, you will have to make some changes and submit for another review

**Required Changes are mandatory changes you need to make before your project can be approved while Optional changes are nice-to-haves or recommendations which would not stop your projects from getting approved.**

## [REQUIRED]
- [x] You should have a descriptive README.md ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
- [x] Add a little margin to the top of your navbar.
- [x] Elements should be aligned with a combination of all three techniques ([float](https://css-tricks.com/all-about-floats/), [flex](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) and [grid](https://css-tricks.com/snippets/css/complete-guide-grid/)). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/flex_float_grid_requirement_pathwright.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/)) and ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/flex_float_grid_requirement_odin.png) from [Odin](https://www.theodinproject.com/courses/html5-and-css3/lessons/positioning-and-floating-elements#introduction)). 
- [x] The Editors' Pick is not well aligned to the right. You should also set a width/max-width to it.
- [x] In **Theory of Inflation section**, the header and the following paragraph are to be aligned to the left, not the center.
- [x] Make sure you add the **suggested newsletter section**
image
- [x] The image in the **Spirals in the sky** section is not of the same width and height as that of the original page.
- [x] In the footer, the section with a class of linked-list should be wider. Increase the width of that section or that of the entire footer.
- [x] There should be 2 horizontal lines right above your footer.
- [x] Add enough margin before your footer.
- [x] In your footer, the first 4 items under the subscribe section are missing the icons. Please add them.
- [x] You should also add some margin before and after the section with a class of **links-ribbon.**

## [OPTIONAL]

- [ ] I noticed that some of the links on your page actually re-direct to the main site. You can just leave the **href empty or add #**.

### Make those changes and submit for another code review.
### Please, make sure you submit the same pull request link. Do not create a new pull request!

## Happy Coding :computer: 

_Feel free to reach out to me on slack_ **`@Elijah Ayandokun`**
  **Elijah O. Ayandokun**
[Github](https://github.com/ElijahTobs) | [LinkedIn](https://www.linkedin.com/in/ayandokunelijah/) | [Twitter](https://twitter.com/ElijahTobs)

**Review by Temmarie — CHANGES_REQUESTED** [review]
### Changes Required! :recycle:
Hi Team ! 👋 👋 

### Welcome to Microverse once again! You have done well so far :clap: However, you did not finish all the previous review changes. Kindly work on them as well as the new ones, and submit them for another code review.

**General**

Please fix the linter issues. Do reach out to me incase you have issues with it.

**Recommended**

- [x] Fix the navbar, you currently have the buttons merging together. Please add some spacings in between.

- [x] You have a horizontal overflow. Please fix this.

- [x] You currently have the "Editors' picks" at a position where it should not be. Please refer to the actual site for clarity.

**Original**

Screenshot (54)

**Yours**

Screenshot (55)

- [x] The images in the "More in space and astronomy" section are lacking some gaps.

**Original**

Screenshot (58)

**Yours**

Screenshot (57)

- [x]  Please fix the footer as well for it to look as similar as possible to the actual sites.

**Original**

Screenshot (48)

**Yours**

Screenshot (59)


__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink:
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

**Review by ebeagusamuel — APPROVED** [review]
# Project Approved :100: 
Hello @jpdf00 and @German-Cobian :wave: 
Nice work on this project. You can now go ahead and merge this branch.
Optionally you can try to improve on the following
- [x] The screenshot in your readme is not loading
- [x] Delete files and folders that you are not making use of from the project, e.g DS_Store.
- [ ] Fix horizontal scroll.

Wish you good luck in the next project.
slack(@ebeagusamuel)

**Pull request merged**
